### PR TITLE
Add FAQs section to homepage

### LINKED ISSUE #71: Move FAQs from documentation onto website
We've discussed previously that we want to move most of the FAQs [from the documentation](https://kedro.readthedocs.io/en/stable/faq/faq.html) onto the website. This is motivated partly because the docs are unwieldy and we need to make FAQs (which are popular) more visible, and also because I don't think they fit in the docs as they are (they're higher level and could be rewritten to have a better marketing slant.

So this ticket is to initiate the discussion/process of creating a new FAQ page. Separately, [I will pick up content revision over on the docs](https://github.com/kedro-org/kedro/issues/1985) to prepare the copy and then you'll have a better idea for what is coming in terms of page design.

### PR DESCRIPTION
**Pull request by tynandebold**
## Description

Resolves #71.

## Development notes

The content for this module is powered by a TypeScript file and each question/answer is an object. To edit the questions, you'll edit that file. There's a slight oddity with how we have to write the answers when we want unordered lists. It isn't that bad and it's quick to pick up that pattern.

## QA notes

Take a look at the deployed URL and check it against the design.

I'm still waiting on the mobile design from @Mackay031. We'll also need to decide on a background color for this section. In the design it was black, but that's the same as the sections above and below the FAQ, which didn't look great, so I've changed it for now.

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Added tests to cover my changes, where applicable

### CODE REVIEW

**Comment by stichbury** [comment]
I really like this, thanks for the preview. I found myself playing around with the sections to see where I could click to expand as it seemed a bit inconsistent as to where and when it opened (it doesn't work on the left hand side of the section, under the question text). I found that a bit odd -- the expand works above the text question but not below? Does that make sense? I can make a small video or show you on a screen share if it helps. But the rest is amazing and this is such a small point.

**Review by stichbury — APPROVED** [review]

**Review by astrojuanlu — APPROVED** [review]
Love this! ⭐

**Comment by tynandebold** [comment]
Nice, thanks for the feedback.

I've updated the accordion hit area and the background colors.

Unfortunately, all the logos have a black background: 

image

Can you help with changing the background to be transparent? All the files are [here](https://github.com/kedro-org/kedro-website/tree/main/public/images/logos).

I'll also look into the transition timing.

**Comment by Mackay031** [comment]
Good point. I could change to background colour of the logos to the dark grey. 

On the timing of the accordion and fade overlap (last point in previous comment, If we think timings of fade will be too tight, could each section of the accordion have a background colour that matches the section background colour (dark grey). This would help in not seeing the fade as the sections move up and down? At least I think it would help :)

**Comment by tynandebold** [comment]
I don't think the background color will do anything. I added an `overflow` property though, and I think that did the trick. See if it looks better to you.

For the images, yeah that would be great. Can you make the background transparent? Wouldn't want us to have to always be changing the color if we change the section background color again in the future.

**Comment by Mackay031** [comment]
Looks great!!

[https://mckinsey.box.com/s/dh2waz3w6n62heoibebydk2mmlzlic8w](url)
The link to .png logos with transparent backgrounds. These are for desktop. Do you use the same images for mobile?

File: components/faq/faq.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            clipRule="evenodd"
+            d="M0 8L8 0L16 8L14.5 9.5L9 4L9 20H7L7 4L1.5 9.5L0 8Z"
+            fillRule="evenodd"
+          />
+        </svg>
+      </motion.div>
+      <AnimatePresence initial={false}>
```
Comment (Huongg) [inline components/faq/faq.tsx:137]: this is very cool, I've never used it before.  I didn't know we use this for the website 😄

**Review by Huongg — APPROVED** [review]
thanks @tynandebold  all working fine for me, couldn't spot any issue 😄

File: components/faq/faq.tsx (hunk 1 above)
Comment (tynandebold) [reply components/faq/faq.tsx:137]: framer-motion you mean?

**Comment by tynandebold** [comment]
Great, thank you @Mackay031. And yes, same logos for desktop and mobile, so no need to do anything else.

Let me know when the mobile designs are ready and I'll implement that.

**Pull request merged**